documentation/cobigen-core_configuration.asciidoc

```diff
@@ -378,6 +378,116 @@ However, if we want to reference an increment that it is not defined inside our
 
 The ref string is split using as delimiter `::`. The first part of the string, is the `trigger_id` to reference. That trigger contains an `increment_id`. Currently, this functionality only works when both templates use the same kind of input file.
 
+== Configuration Upgrade
+
+`CobiGen 2021.12.007` introduces an updated configuration folder structure:
+
+```
+template-sets
+ |- downloaded
+   |- template-set1.jar
+   |- template-set2.jar
+   |- template-set-utilities.jar
+ |- adapted
+   |- template-set1
+     |- src
+       |- main
+         |- templates
+           |- templates
+             |- context.xml
+             |- templates.xml
+   |- template-set2
+```
+
+The updated structure splits the context.xml and the pom.xml files into multiple directories - one for each template set
+ - each with their own atomized/small/more understandable version so you can download them individually instead of all
+ at once
```

Comment:
Can you give some information that the upgrade can be done automatically and that the user will be asks if he wants to upgrade, and he can postpone the question. I think we should also recommend the automatic upgrade.